.github/actions/schema-validator/index.js

```diff
@@ -0,0 +1,59 @@
+const core = require('@actions/core');
+const dirtyJSON = require('dirty-json');
+const Validator = require('jsonschema').Validator;
+const inputData = core.getInput('INPUT_DATA') || process.env.INPUT_DATA;
+
+const nowSecureIssueSchema = {
+    "type": "object",
+    "patternProperties": {
+        "^[a-f0-9]{8}-[a-f0-9]{4}-4[a-f0-9]{3}-[89aAbB][a-f0-9]{3}-[a-f0-9]{12}$": {
+            "type": "object",
+            "properties": {
+                "key": {
+                    "type": "string"
+                },
+                "title": {
+                    "type": "string"
+                },
+                "description": {
+                    "type": "string"
+                },
+                "recommendation": {
+                    "type": "string"
+                },
+                "severity": {
+                    "type": "string"
+                },
+                "assessmentVersion": {
+                    "type": "string"
+                },
+                "platform": {
+                    "type": "string"
+                }
+            },
+            "required": ["key", "title", "description", "severity", "assessmentVersion", "platform"]
```

Comment:
What happens if a user has decided to add in the output some extra fields coming from NowSecure? We provide the functionality as described [here](https://github.com/camelotls/actions-nowsecure/blob/d752b264ece18b74be4cdbe0ef5ff36b526c09c9/index.js#L137-L154).